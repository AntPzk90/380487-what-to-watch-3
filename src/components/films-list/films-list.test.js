import React from 'react';
import renderer from 'react-test-renderer';
import FilmsList from './films-list.jsx';

const films = [
  {
    title: `Bohemian rhapsody`,
    src: `bohemian-rhapsody.jpg`,
    poster: `bohemian-rhapsody.jpg`,
    titlePoster: `bohemian-rhapsody.jpg`,
    genre: `Comedy`,
    releaseDate: `2015`,
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
];

const genreToFilter = `All genres`;

it(`SnapshotTest FilmList`, () => {
  const tree = renderer
  .create(<FilmsList
    films = {films}
    showCardOverview = {() => {}}
    genreToFilter = {genreToFilter}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
