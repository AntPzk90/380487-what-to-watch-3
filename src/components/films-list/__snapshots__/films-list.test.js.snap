// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest FilmList 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
      onClick={[Function]}
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="https://htmlacademy-react-3.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        War of the Worlds
      </a>
    </h3>
  </article>
</div>
`;
